manyfaced/common/bearstorage.py

```diff
@@ -30,21 +31,22 @@ def __init__(self, ip, raw_request, timestamp, parsed_request, is_detected,
                 self.ua = parsed_request.headers['user-agent']
         self.isDetected = is_detected
         self.hostname = hostname
-        location = geolite2.lookup(ip)
-        if location is not None:
-            self.country = location.country
-            self.continent = location.continent
-            self.timezone = location.timezone
+        #location = geolite2.lookup(ip)
+        #if location is not None:
+        #    self.country = location.country
+        #    self.continent = location.continent
+        #    self.timezone = location.timezone
```

Comment:
You can uncomment this and use python-geoip-python3 library with exact same interface